### PR TITLE
add user-friendly error msg

### PR DESCRIPTION
**Pull request by bjfresh**
Clearer error message in Place Bid flow

### CODE REVIEW

File: @shared/utils/formatContractError.ts [hunk 1]
```diff
@@ -35,6 +35,8 @@ export function formatContractError(error: TransactionError | Error) {
   }
 
   switch (error?.message) {
+    case 'Connector not found':
+      return 'You must connect your wallet to place a bid'
```
Comment (almndbtr) [inline @shared/utils/formatContractError.ts:39]: Please forgive me with the drive-by suggestion! 🚗 

To stay consistent with its sibling message, it would be nice to end this sentence with punctuation:

```suggestion
      return 'You must connect your wallet to place a bid.'
```

**Review by olegakbarov — APPROVED** [review]
